jetty-ee10/jetty-ee10-tests/jetty-ee10-test-log4j2-webapp/pom.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.eclipse.jetty.ee10</groupId>
+    <artifactId>jetty-ee10-tests</artifactId>
+    <version>12.0.9-SNAPSHOT</version>
+  </parent>
+  <artifactId>jetty-ee10-test-log4j2-webapp</artifactId>
+  <packaging>war</packaging>
+  <name>EE10 :: Tests :: Log4j2 WebApp</name>
+
+  <properties>
+    <bundle-symbolic-name>${project.groupId}.log4j2</bundle-symbolic-name>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-core</artifactId>
+      <version>2.20.0</version>
```

Comment:
```suggestion
      <version>${log4j2.version}</version>
```

Reply:
If log4j changes their implementation, this might render the point of this webapp moot.
I'll change it to use the property.

Reply:
oh right. Sorry I didn't pay attention to this